packages/dnb-eufemia/src/components/accordion/Accordion.tsx

```diff
@@ -228,6 +229,15 @@ function Accordion({
     }
   }, [context.flush_remembered_state, context.expanded_id])
 
+  // Add callback for closing all accordions inside a group
+  if (
+    !hasAddedCallbackRef.current &&
+    context?.collapseAccordionCallbacks
```

Comment:
```suggestion
    context?.collapseAccordionCallbacks &&
    !hasAddedCallbackRef.current
```
If the first does not match the second don't needs to run. When opposite, the second needs to run always.

Reply:
Great point 👍